www/templates/html/mh-sort-filter.tpl.php

```diff
@@ -0,0 +1,23 @@
+<nav class="mh-sort-filter" aria-labelledby="mh-sort-filter-<?php echo $context->group_id; ?>">
+    <span id="mh-sort-filter-<?php echo $context->group_id; ?>" class="sort-filter-title">Order By:</span>
+    <ul class="mh-btn-group">
+
+        <?php foreach ($context->buttons as $key=>$details):?>
+            <?php
+            $is_current_option = ($context->selected_key == $key) ? true : false;
+            $active_class = $is_current_option ? 'active' : '';
+            ?>
+            <li>
+                <a href="<?php echo htmlentities($details['url'], ENT_QUOTES) ?>" class="wdn-button wdn-button-brand <?php echo $active_class ?>">
```

Comment:
You are not wrong, and I would like to see this fixed. However I think it is out of scope of the current PR. Savvy in the project has never been configured to escape... and I'm worried about double encoding if I turn it own now without being super careful.